client/components/AllProducts.js

```diff
@@ -43,12 +43,14 @@ class AllProducts extends React.Component {
             ? filteredProducts.map(product =>
               (<li key={product.id}>
                 <Link to={`/products/${product.id}`}>{product.title}</Link>
```

Comment:
why did you remove the add to cart button? The requirements said that we should be able to add a product to the cart from the products page and in the single product view...